### PR TITLE
Further clean-up of helper scripts

### PR DESCRIPTION
**Pull request by jmurty**
Make further improvements and clean up helper scripts, following the consolidation of these scripts into the single *transcrypt* owner script as of 561d158bb79fff95b1c37b195266990c97e6a6c3

### CODE REVIEW

File: .github/workflows/run-bats-core-tests.yml [hunk 1]
```diff
@@ -40,6 +40,13 @@ jobs:
     - name: Print OpenSSL version
       run: openssl version
 
+    - name: Print Git version
+      run: git version
+
+    # Configure default Git branch name to suppress hint warnings
+    - name: Configure default Git branch to "main"
+      run: git config --global init.defaultBranch main
```
Comment (elasticdog) [inline .github/workflows/run-bats-core-tests.yml:48]: I'd be glad to rename our default branch to `main` for the transcrypt repository itself as well...I'm not currently aware of anyone writing packages based on pulling `master`, so it shouldn't be terribly disruptive (and GitHub will let users know how to update their local clones).

Comment (elasticdog) [reply .github/workflows/run-bats-core-tests.yml:48]: We would have to update our GitHub Actions workflow to match the new branch name, but I think that would be it?

Comment (jmurty) [reply .github/workflows/run-bats-core-tests.yml:48]: Yes, let's do it! I think I have covered the necessary changes in https://github.com/elasticdog/transcrypt/pull/122

Comment (elasticdog) [reply .github/workflows/run-bats-core-tests.yml:48]: I've merged in #122 and updated the branch, but the tests will fail until we get this `defaultBranch` setting in place...I should have thought a bit more about the order of operations here, but not a huge deal.

**Review by elasticdog — APPROVED** [review]
I think you'll have to rebase to get the tests passing again (and deal with the merge conflicts), but otherwise this is looking good to me. I'm excited that all of the recent refactoring is making the project more testable and resilient! Thanks @jmurty!

**Comment by jmurty** [comment]
This work is now squashed and merged to `main` in fdf81c53f0ad27651e03a67ea732b164d209e948

**Pull request closed without merging**